content/courses/tokens-and-nfts/token-program.md

```diff
@@ -852,7 +869,7 @@ approval.
 2. Create a new Token Mint in the `CreateMint` component. If you need a
    refresher on how to send transactions to a wallet for approval, have a look
    at the
-   [Wallets lesson](/content/courses/intro-to-solana/interact-with-wallets.md).
+   [Wallets lesson](/developers/courses/intro-to-solana/interact-with-wallets).
```

Comment:
It's weird but links should have .md associated with them - see https://github.com/solana-foundation/developer-content/blob/main/CONTRIBUTING.md#links

Do that for all links.